fvm/blueprints/contracts.go

```diff
@@ -59,3 +59,49 @@ func DeployContractTransaction(address flow.Address, contract []byte, contractNa
 		SetScript([]byte(fmt.Sprintf(deployContractTransactionTemplate, contractName, hex.EncodeToString(contract)))).
 		AddAuthorizer(address)
 }
+
+const auditContractForDeploymentTransactionTemplate = `
```

Comment:
This is used only in tests, so I would prefer this being in a test module instead of in blueprints directly.